src/pages/tickets/org/orgHeader/index.tsx

```diff
@@ -266,14 +330,61 @@ export const OrgHeader = ({
               <Icon src={searchIcon} alt="Search" />
             </SearchWrapper>
           </FiltersRight>
-          <FiltersLeft>
-            <SoryByContainer>
-              <FilterLabel>Sort by:Newest First</FilterLabel>
-              <DropDownButton>
-                <Img src={dropdown} alt="" />
-              </DropDownButton>
-            </SoryByContainer>
-          </FiltersLeft>
+          <EuiPopover
```

Comment:
Hi @okhot,
Please, @okhot, review your previous code changes and also consider examining other files if necessary to ensure everything is aligned and functioning as expected